ocs_ci/ocs/resources/bucketclass.py

```diff
@@ -165,12 +165,18 @@ def _create_bucket_class(bucket_class_dict):
                     namespace_policy["write_resource"] = namespacestores[0].name
 
         elif "backingstore_dict" in bucket_class_dict:
-            backingstores = [
-                backingstore
-                for backingstore in backingstore_factory(
-                    interface, bucket_class_dict["backingstore_dict"], timeout=timeout
-                )
-            ]
+            try:
+                backingstores = [
+                    backingstore
+                    for backingstore in backingstore_factory(
+                        interface,
+                        bucket_class_dict["backingstore_dict"],
+                        timeout=timeout,
+                    )
+                ]
+            except TimeoutExpiredError:
```

Comment:
we don't know if cloud provider is slow, no? The log message could be,
'Backingstore was not created after <> seconds'